src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -148,5 +184,78 @@ public Contig apply(String contigString) {
 
     return contigs;
   }
+  
+  /**
+   * Get a list of Contigs representing sharded windows of all genomic regions within the variant set.
+   * 
+   * These are useful for pipelines that operate in parallel against data within a specific variant set.
+   * 
+   * @param genomics - the genomics client
+   * @param variantSetId - the id of the variant set for which to create shards
+   * @param sexChromosomeFilter - whether or not to include the sex chromosomes in the list of shards
+   * @return a shuffled list of shards
+   * @throws IOException
+   */
+  public static List<Contig> getAllShardsInVariantSet(final Genomics genomics, final String variantSetId,
+      final SexChromosomeFilter sexChromosomeFilter) throws IOException {
+    return getAllShardsInVariantSet(genomics, variantSetId, sexChromosomeFilter, DEFAULT_NUMBER_OF_BASES_PER_SHARD); 
+  }
+  
+  /**
+   * Get a list of Contigs representing sharded windows of all genomic regions within the variant set.
```

Comment:
This doesn't tell the whole story - unless I'm mistaken it looks like those contigs start at 0, instead of starting at the variant's start position (line 182). So this isn't actually a simple typecast from Variant to bounding interval.